### ISSUE TITLE
`make build` is broken

### ISSUE THREAD

**Issue by xni** [pypa/warehouse#4939]
```
  Could not find a version that satisfies the requirement itsdangerous==1.0.0 (from -r /tmp/requirements/main.txt (line 238)) (from versions: 0.9, 0.9.1, 0.10, 0.11, 0.12, 0.13, 0.14, 0.15, 0.16, 0.17, 0.18, 0.19, 0.20, 0.21, 0.22, 0.23, 0.24, 1.1.0)
No matching distribution found for itsdangerous==1.0.0 (from -r /tmp/requirements/main.txt (line 238))
ERROR: Service 'web' failed to build: The command '/bin/sh -c set -x     && PIP_EXTRA_INDEX_URL=$THEME_REPO         pip --no-cache-dir --disable-pip-version-check             install --no-binary hiredis                     -r /tmp/requirements/deploy.txt                     -r /tmp/requirements/main.txt                     $(if [ "$DEVEL" = "yes" ]; then echo '-r /tmp/requirements/tests.txt'; fi)                     $(if [ "$THEME_REPO" != "" ]; then echo '-r /tmp/requirements/theme.txt'; fi)     && find /opt/warehouse -name '*.pyc' -delete' returned a non-zero code: 1
make: *** [build] Error 1
```

**Comment by xni** [comment]
Looks like the problem is that `itsdangerous=1.0.0` was removed

**Comment by di** [comment]
Fixed in #4940.
